validator/README.md

```diff
@@ -0,0 +1,74 @@
+## OpenTelemetry SDK Configuration Validator
+
+This application will replace environment variables in values of valid yaml or
+json files, following the rules of [file configuration environment variable
+substitution](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/file-configuration.md#environment-variable-substitution),
+before validating that result against the [OpenTelemetry SDK Configuration
+schema](https://github.com/open-telemetry/opentelemetry-configuration/).
+
+### Build
+
+The `schema` directory is required to be in the directory of the Go file that
+embeds it, so a `go build` alone will fail, instead run `make` which will copy
+the schema:
+
+```
+$ make validator
+```
+
+Same is true for building the docker image:
+
+```
+$ make validator-docker
+```
+
+### Usage
+
+The command `otel_config_validator` takes one argument, the path to the yaml or
+json configuration file and optionally the path to a file to output the
```

Comment:
Oh. I just assumed since it was defined as a jsonschema it meant json was supported.

I'm fine removing the json input support (except for the work I put into it, haha, but that is my fault for misunderstanding). 

However, what about json output? The reason I want json output is that it is more often found in standard libraries (that is the case now in Erlang/Elixir).

Reply:
I don't have any problem with the program outputing a JSON representation as an intermediate step in the parsing process. Users should just be informed that as of now, the only file format they should use is YAML.

Reply:
Ok, json input support has been removed.